### PR TITLE
Add fake capture on new braspag api

### PR DESCRIPTION
**Pull request by antoniofilho**

### CODE REVIEW

**Comment by no-donut-for-you** [comment]
:family: :gun:

File: lib/fake_braspag/sales.rb [hunk 1]
```diff
@@ -14,6 +14,14 @@ class Sales < Sinatra::Base
       end
     end
 
+    put '/:PaymentId/capture' do
+      if ResponseToggler.enabled?('capture')
+        jbuilder :sales_capture
+      else
+        jbuilder :sales_capture_failure
+      end
```
Comment (pmatiello) [inline lib/fake_braspag/sales.rb]: A suggestion:

```ruby
jbuilder(ResponseToggler.enabled?('capture') ? :sales_capture : :sales_capture_failure)
```

**Comment by pmatiello** [comment]
:sheep:

**Comment by rizidoro** [comment]
:fire: :newspaper:

**Pull request merged**